freqtrade/analyze.py

```diff
@@ -44,13 +44,13 @@ def parse_ticker_dataframe(ticker: list) -> DataFrame:
         :param ticker: See exchange.get_ticker_history
         :return: DataFrame
         """
-        cols = ['date', 'open', 'high', 'low', 'close', 'volume']
```

Comment:
I'm perfectly fine having backtesting break for now. The way forwad is the `ccxt` way. We can think of creating a converter that goes through the data and transforms the old data to the new format, or then for backtesting add an temporary intermediate that overwrites the `parse_ticker_dataframe` to use the old format